pkg/endpoint/bpf.go

```diff
@@ -1089,12 +1089,8 @@ func (e *Endpoint) updatePolicyMapPressureMetric() {
 // not otherwise changed (e.g., it is never set to 'false').
 func (e *Endpoint) deletePolicyKey(keyToDelete policy.Key, incremental bool, hadProxy *bool) bool {
 	// Convert from policy.Key to policymap.Key
-	policymapKey := policymap.PolicyKey{
-		Identity:         keyToDelete.Identity,
-		DestPort:         keyToDelete.DestPort,
-		Nexthdr:          keyToDelete.Nexthdr,
-		TrafficDirection: keyToDelete.TrafficDirection,
-	}
+	policymapKey := policymap.NewKey(keyToDelete.Identity, keyToDelete.DestPort,
```

Comment:
nit: Please revert the unintended alignment changes. Same for lines below.

Reply:
This change is more than an alignment change, it is a change from a struct to a function call. While member per line makes sense for a struct initialization as it mirrors the struct definition, there is no such correspondence for function call. We typically break up function call paramaters to different lines if lines get long.

Reply:
So the change here (and below) was intended.

Reply:
same here.